### ISSUE TITLE
[Feat] Add Favicon feature

### ISSUE THREAD

**Issue by furkankurt** [unfoldadmin/django-unfold#415]
Hello,

Django Unfold doesn't have a favicon feature.
We had to override the whole `skeleton.html` template to add a favicon. (already mentioned in #206)

I have added a `site_favicon.html` file to the `unfold/helpers` directory, we can add all favicon types to the website.
Instead of overriding the big `skeleton.html`, we will override just the `site_favicon.html`.

I've already created the PR #382.
I'm excited to see you merging it.

**Comment by lukasvinclav** [comment]
Added #560

**Comment by danieltsega** [comment]
**Comment:**

Hello,

I followed the steps mentioned in #382 to add a favicon to the Django admin interface while using `django-unfold`. However, I'm still unable to see the favicon in the admin pages.

Here’s what I did:
1. Created the `helpers` directory under the `unfold/` package (as suggested).
2. Added the `site_favicon.html` file inside the `helpers` folder with the necessary `<link>` tags to include the favicon files.
3. Placed the favicon files in the `static/favicon/` directory.
4. Restarted the Django server.

Despite following these steps, the favicon still doesn’t appear on the admin interface. It works fine on the other pages of the website, but not for the Django admin.

Is there something I might have missed, or could there be an additional step required to make it work specifically with `django-unfold`? Any guidance would be much appreciated.

Thank you!
